src/styles/components/badges/styles.less

```diff
@@ -16,163 +16,167 @@ components/badges.less
 /* -----------------------------------------------------------------------------
 --------------------------------------------------------------------------------
 
-Badge
+Badges
 
 --------------------------------------------------------------------------------
 ----------------------------------------------------------------------------- */
 
-.badge {
+& when (@badges-enabled) {
 
-  color: @body-background-color;
-  font-size: @body-text-font-size;
-  line-height: @body-text-line-height;
-  padding: 0 @base-spacing-unit * 0.25;
```

Comment:
Not sure what our position is on this stylistically, but for arithmetic like this, I'd aim to place them in explicit parentheses.